doc/sphinx/practical-tools/utilities.rst

```diff
@@ -787,6 +790,13 @@ of your project.
 
        (include_subdirs qualified)
 
+   The ``.v`` file that will load the plugin, needs to use the
+   following special syntax for compatibility with current Dune versions:
```

Comment:
```suggestion
   For now, each ``.v`` file that refers to the plugin, must include following command for
   compatibility with current Dune versions:
```

I suppose it has to be each .v file referencing the plugin because there's no way to know which one is loaded first.